config/charts/knative-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: knative-operator
-version: {{ version }}
+version: 1.10.1
```

Comment:
Why did you pin a specific version? I think we would like to release the charts based on different versions.